dockers/docker-dhcp-server/cli-plugin-tests/mock_config_db.json

```diff
@@ -15,16 +15,10 @@
         "value": "dummy_value"
     },
     "DHCP_SERVER_IPV4_RANGE|range1": {
-        "ranges": [
-            "100.1.1.3",
-            "100.1.1.5"
-        ]
+        "range": "100.1.1.3,100.1.1.5"
     },
     "DHCP_SERVER_IPV4_RANGE|range2": {
-        "ips": [
-            "100.1.1.7",
-            "100.1.1.8"
-        ]
+        "range": "100.1.1.9,100.1.1.8"
```

Comment:
General comment, but would it be better to have separate `start` and `end` entries here, so that it's clear to the user what the semantics are, and processing the values here is a bit easier?

Reply:
In previous HLD review, Guohan suggested to simplify this field so that we can support 1 ip configured in range, if range list only contains 1 ip, we treat it as a range only have 1 ip with same start and end.

Reply:
> > General comment, but would it be better to have separate `start` and `end` entries here instead of a single `range` entry, so that it's clear to the user what the semantics are, and processing the values here is a bit easier?
> 
> @yaqiangz could you help reply?

Have replied, pls see above.